test_setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# this one is to get everything set up without a hitch so you can run your
+# ansible command
+
+USERNAME=$USER
+
+# get ansible-conda down if it is not already
```

Comment:
"get ansible-conda down"? what does that mean? It should be "Update the ansible-conda to the latest version." to be more clear.

This is not a good practice, updating to the latest version. You need to pin the update to the specific commit in that repo and update that commit hash only after testing.

Reply:
I believe when you run `git submodule update`, the submodule already updates to the specific commit tagged in the repository.

Reply:
fixed at 6206f4015d63ef6bd96bcc302f07a6145713476c